source/Element/Component.h

```diff
@@ -5,9 +5,9 @@ class Component
 {
 public:
 	Component(Entity* anOwner) : myOwner(anOwner) {};
```

Comment:
I'm pretty sure I should make these virtual.

Reply:
No? You don't made constructors virtual.